mlflow/ml-package-versions.yml

```diff
@@ -272,3 +272,21 @@ statsmodels:
     maximum: "0.12.2"
     run: |
       pytest tests/statsmodels/test_statsmodels_autolog.py --large
+
+spark:
+  package_info:
+    pip_release: "pyspark"
+
+  models:
+    minimum: "3.0.0"
+    maximum: "3.1.1"
+    requirements: ["boto3", "scikit-learn", "pyarrow"]
+    run: |
+      SAGEMAKER_OUT=$(mktemp)
+      if mlflow sagemaker build-and-push-container --no-push --mlflow-home . > $SAGEMAKER_OUT 2>&1; then
+        echo "Sagemaker container build succeeded.";
+      else
+        echo "Sagemaker container build failed, output:";
+        cat $SAGEMAKER_OUT;
```

Comment:
The image built here is used in `tests/spark/test_spark_model_export.py`.

Reply:
if Sagemaker container build failed, shall we skip running tests/spark?

Reply:
Yeah I'll add `exit 1`

Reply:
Done!